public/landingPage.html

```diff
@@ -85,14 +85,14 @@
         <div id="background-overlay" class="hidden"></div>
         <div id="main-view-container" class="container hidden">
             <div id="main-view" ui-view="content">
-                <h3 id="title">PenPal Gladiators</h3>
+                <h3 id="title">PenPal Gladiator Project</h3>
                 <p id="tag">
                     Politics is a complicated topic. People have disagreements and it's hard to agree on something. But what if you could talk about it? To be able to discuss differences with someone of opposing opinion?
                 </p>
                 <p>
-                    Find people of other perspectives and learn something new.
+                    Find people of other perspectives and learn something new in the Coliseum.
```

Comment:
This will also be replaced (in theory) by the text in `landing-overhaul` (I think that's the name of my branch)